src/clj/kixi/hecuba/data/measurements.clj

```diff
@@ -204,7 +204,16 @@
    Returns a list of maps, with all values parsed approprietly."
   [measurements]
   (map (fn [m] (assoc-in m [:value]
-                         (let [n (edn/read-string (:value m))]
+                         (let [n (if (number? (:value m)) 
+                                   (:value m)
+                                   (try 
+                                     (Double/valueOf (:value m))
+                                     (catch NumberFormatException e nil
+                                            (log/error e
+                                                       "> NumberFormatException in parse-measurements"))
+                                     (catch NullPointerException e nil
+                                            (log/error e
+                                                       "> NumberFormatException in parse-measurements"))))] 
```

Comment:
Would be good if we could log which sensor id and date was causing the issue. I *think* this is available here.